libqtile/layout/floating.py

```diff
@@ -268,3 +278,19 @@ def cmd_next(self):
     def cmd_previous(self):
         # This can't ever be called, but implement the abstract method
         pass
+
+
+class FloatingTile(Floating):
+    '''Floating layout for tile layouts config: the `layouts` symbol in config.py.
+
+    It is almost exact with Floating, except that it's intended to be use a tile layout.'''
```

Comment:
I'm not clear on what this actually does or why we need it. Is it basically another layout on top of the existing layout?